### DISCUSSION TITLE
Bug: List of packages for my account (~broofa) shows package I don't actually maintain

### DISCUSSION THREAD

**Discussion by broofa** [npm/feedback#857] [Website] [answered]
* Navigate to https://www.npmjs.com/~broofa
* Note that it lists `random-uuid` as one of the packages I maintain
* Click on that package
* Notice that the package is actually maintained by `~bubkoo`, and is _definitely not a package I maintain_.

I do, however, maintain the similarly named `randomuuid` package (no hyphen).  So possibly related to that somehow?

**Comment by MylesBorins** [comment]
Thanks for sharing @broofa I've asked engineering to look into this. Please also consider opening a ticket with support when you notice issues like this at https://npmjs.com/support

**Comment by MylesBorins** [comment, marked as answer]
It appears that ownership was given to you as part of the naming dispute for the randomuuid package in early 2021. I've gone ahead and removed that ownership. Thanks for letting us know
